### PR TITLE
Preference managers

### PR DESCRIPTION
**Pull request by NicoleParmentier**
Pulled from Cultivar and improved by splitting different preference sections into classes. Plus some tidying of the GBA pallette.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,6 +29,19 @@
 				<artifactId>maven-surefire-plugin</artifactId>
```
Comment (NicoleParmentier) [inline pom.xml:29]: Increment Version plz

File: src/com/zalinius/libgdxtools/graphics/ColorTools.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import com.badlogic.gdx.graphics.Color;
+
+public class ColorTools {
+	public static final int MIN_COLOR_CHANNEL_VALUE = 0;
+	public static final int MAX_COLOR_CHANNEL_VALUE = 255;
+
+	/**
+	 *
+	 * @param red The red color channel value, must be within the range [0, 255]
+	 * @param green The green color channel value, must be within the range [0, 255]
```
Comment (NicoleParmentier) [inline src/com/zalinius/libgdxtools/graphics/ColorTools.java:12]: Rename the test class for this plz

File: src/com/zalinius/libgdxtools/tools/Assets.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import com.badlogic.gdx.audio.Music;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas.AtlasRegion;
+import com.badlogic.gdx.utils.Array;
+
+public class Assets {
+
```
Comment (NicoleParmentier) [inline src/com/zalinius/libgdxtools/tools/Assets.java:23]: This is Zach's problem now apparently

File: src/com/zalinius/libgdxtools/tools/Assets.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+	private Assets() {}
+
```
Comment (NicoleParmentier) [inline src/com/zalinius/libgdxtools/tools/Assets.java:25]: Rename this from Assets

**Review by NicoleParmentier — COMMENTED** [review]
Also, I need you to make a jenkinsfile change so it runs your Integration Tests

**Review by Zalinius — APPROVED** [review]
Beautiful *chef's kiss*

**Pull request merged**